protostar/commands/cairo1_commands/cairo1_build_command.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from pathlib import Path
+from typing import List, Optional, Any
+
+from protostar.cli import ProtostarArgument, ProtostarCommand
+from protostar.cli.common_arguments import COMPILED_CONTRACTS_DIR_ARG
+from protostar.compiler import (
+    ProjectCompiler,
+)
+import protostar.cairo.cairo_bindings as cairo1
+
+
+class Cairo1BuildCommand(ProtostarCommand):
+    def __init__(
+        self,
+        project_compiler: ProjectCompiler,
+    ):
+        super().__init__()
+        self._project_compiler = project_compiler
```

Comment:
what we really need here is only `configuration_file`, not a legacy project compiler

Reply:
i think this point still stands

Reply:
I think there are two comments about the same thing, let's follow up in https://github.com/software-mansion/protostar/pull/1551#discussion_r1127040435